metabrainz/templates/payments/donate.html

```diff
@@ -105,39 +105,39 @@ <h1 class="page-title">Make a Donation</h1>
 
     <p>
       <small>
-        The personal information provided to the MetaBrainz Foundation during the
+        {{ _('The personal information provided to the MetaBrainz Foundation during the
         donation process will not be shared with anyone. For more information,
-        please take a look at our <a href="{{ url_for('index.privacy_policy') }}">
-        privacy policy</a>.
+        please take a look at our <a href="%(url)s">
+        privacy policy</a>.', url=url_for('index.privacy_policy')) }}
       </small>
     </p>
 
     <p>
       <small>
-        To find out how to cancel recurring donations take a look at
-        <a href="{{ url_for('payments.cancel_recurring') }}">this page</a>.
+        {{ _('To find out how to cancel recurring donations take a look at
+        <a href="%(url)s">this page</a>.', url=url_for('payments.cancel_recurring')) }}
       </small>
     </p>
 
-    <h2>Other ways to donate</h2>
+    <h2>{{ _('Other ways to donate') }}</h2>
 
-    <h3>Non-US Bank transfer</h3>
-    <p>You can make a donation, or recurring donations, via bank transfer to the following IBAN:</p>
+    <h3>{{ _('Non-US Bank transfer') }}</h3>
+    <p>{{ _('You can make a donation, or recurring donations, via bank transfer to the following IBAN:') }}</p>
     <p><em style="margin-left: 2em">ES39 0182 6536 7602 0155 8258 (BIC: BBVAESMMXXX)</em></p>
-    <p>Please indicate your name and use the word <em>DONATION</em> in the description field.</p>
+    <p>{{ _('Please indicate your name and use the word <em>DONATION</em> in the description field.') }}</p>
 
-    <h3>US Check</h3>
-    <p>If you would like to donate using a check drawn on a US bank, please send them to:</p>
+    <h3>{{ _('US Cheque') }}</h3>
+    <p>{{ _('If you would like to donate using a cheque drawn on a US bank, please send them to:') }}</p>
```

Comment:
Please don't modify strings. "Check" is a correct word for this in american english.